sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -5321,21 +5321,25 @@ static void pi2urImageCopyFlags(const pi_image_copy_flags PiFlags,
   }
 }
 
-inline pi_result
-piextMemImageCopy(pi_queue Queue, void *DstPtr, void *SrcPtr,
-                  const pi_image_format *ImageFormat,
-                  const pi_image_desc *ImageDesc,
-                  const pi_image_copy_flags Flags, pi_image_offset SrcOffset,
-                  pi_image_offset DstOffset, pi_image_region CopyExtent,
-                  pi_image_region HostExtent, pi_uint32 NumEventsInWaitList,
-                  const pi_event *EventWaitList, pi_event *Event) {
+inline pi_result piextMemImageCopy(
+    pi_queue Queue, void *DstPtr, void *SrcPtr,
+    const pi_image_desc *SrcImageDesc, const pi_image_desc *DestImageDesc,
+    const pi_image_format *SrcImageFormat,
+    const pi_image_format *DestImageFormat, const pi_image_copy_flags Flags,
+    pi_image_offset SrcOffset, pi_image_offset DstOffset,
+    pi_image_region CopyExtent, pi_uint32 NumEventsInWaitList,
+    const pi_event *EventWaitList, pi_event *Event) {
   PI_ASSERT(Queue, PI_ERROR_INVALID_QUEUE);
 
   auto UrQueue = reinterpret_cast<ur_queue_handle_t>(Queue);
 
-  ur_image_format_t UrFormat{};
-  ur_image_desc_t UrDesc{};
-  pi2urImageDesc(ImageFormat, ImageDesc, &UrFormat, &UrDesc);
+  ur_image_format_t UrFormatSrc{};
+  ur_image_desc_t UrDescSrc{};
+  pi2urImageDesc(SrcImageFormat, SrcImageDesc, &UrFormatSrc, &UrDescSrc);
+
+  ur_image_format_t UrFormatDest{};
+  ur_image_desc_t UrDescDest{};
+  pi2urImageDesc(DestImageFormat, DestImageDesc, &UrFormatDest, &UrDescDest);

```

Comment:
Please use `Dst` throughout to be consistent (currently have mix of `Dest` and `Dst`).

Additionally, variables in the style of `Ur__Src` and `Ur__Dest` should be renamed to `UrSrc__` and `UrDst__` for consistency with existing variables.

Reply:
applied renaming.